stable/instana-agent/templates/daemonset.yaml

```
@@ -0,0 +1,164 @@
+{{- if .Values.agent.key -}}
+{{- if .Values.zone.name -}}
+apiVersion: apps/v1beta2
+kind: DaemonSet
+metadata:
+  name: {{ template "instana-agent.name" . }}
+  labels:
+    {{- include "instana-agent.helmChartLabels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "instana-agent.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "instana-agent.helmChartLabels" . | nindent 8 }}
+      {{- if .Values.agent.pod.annotations }}
+      annotations:
+        {{ toYaml .Values.agent.pod.annotations | nindent 8}}
+      {{- end }}
+    spec:
+      serviceAccount: {{ template "instana-agent.serviceAccountName" . }}
+      hostIPC: true
+      hostNetwork: true
+      hostPID: true
+      containers:
+        - name: {{ template "instana-agent.name" . }}
+          image: "{{ .Values.agent.image.name }}:{{ .Values.agent.image.tag }}"
+          imagePullPolicy: {{ .Values.agent.image.pullPolicy }}
+          env:
+            - name: INSTANA_AGENT_LEADER_ELECTOR_PORT
+              value: {{ int .Values.agent.leaderElectorPort | quote }}
```

It should be necessary to use the `int` function here.
```suggestion
              value: {{ .Values.agent.leaderElectorPort | quote }}
```

---

I think you mean: "should _not_ be necessary" ?